README.md

```diff
@@ -77,50 +77,40 @@ Clone this repository to your desired folder:
 
 Install this project with:
 
-<!--
 Example command:
 
 ```sh
   cd my-project
-  gem install
+  sudo gem install ruby
+  sudo gem install rails
 ```
---->
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
   rails server
 ```
---->
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
-
 ```sh
-  bin/rails test test/models/article_test.rb
+  bin/rails test test/models/comment_test.rb
+  bin/rails test test/models/user_test.rb
+  bin/rails test test/models/post_test.rb
+  bin/rails test test/models/like_test.rb
 ```
---->
 
 ### Deployment
 
 You can deploy this project using:
 
-<!--
-Example:
-
 ```sh
-
+  Not yet deployed
 ```
```

Comment:
- [ ] Please your MIT license is broken. Kindly set it up. we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.